apps/frontend/src/pages/access-management/libs/constants/column-header.constant.ts

```diff
@@ -0,0 +1,10 @@
+import { type ColumnDef } from "@tanstack/react-table";
+
+import { type User } from "../types/types.js";
+
+const columnHeader: ColumnDef<User>[] = [
```

Comment:
```suggestion
const TABLE_COLUMNS: ColumnDef<User>[] = [
```
don't forget about file name